src/components/VHeader/VFilterButton.vue

```diff
@@ -0,0 +1,124 @@
+<template>
+  <VButton
+    variant="action-menu"
+    class="self-center"
+    :pressed="sidebarVisible"
+    @click="toggleFilters"
+  >
+    <FilterIcon v-show="showIcon" class="w-6 h-6" width="24" height="24" />
```

Comment:
It's nice to be able to inline icons like this, but should we be using the `VIcon` component instead? It seemed nice to be able to remove the size classes and attribute duplication using it.